app/konnect/geo.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Geographic Regions
+content_type: explanation
+no_version: true
+---
+
+{{site.konnect_saas}} allows you to host and operate your cloud instance in a geographic region that you specify. This is important for data privacy and regulatory compliance for you organization. 
+
+Geographic regions allow you to also operate {{site.konnect_saas}} in a similar geo to your users and their infrastructure applications. 
+<!--- Do not publish yet: "This reduces network latency and minimizes the blast-radius in the event of cross-region connectivity failures." -->
+
+{:.note}
+> **Note:** Only Enterprise plans can operate in more than one geo at a time. For more information about how to upgrade your plan, see [Change to a Different Plan](/konnect/account-management/).
```

Comment:
This is no longer true. All Konnect customers regardless of tier will have access to all Konnect geos out of the box.